MessengerOnFileForLinux/Common/src/TimeManagment.cpp

```diff
@@ -0,0 +1,34 @@
+#include "TimeManagment.hpp"
+
+#include <ctime>
+#include <algorithm>
+
+namespace
+{
+void removeEmptyCharactersFromTimeEnd(std::string& date)
+{
+   date.pop_back();
+   date.pop_back();
+}
+}
+
+namespace TimeManagment
```

Comment:
It is the same. It doesn't matter that if use namespace x { fucntion()} or x::function().  Which option will we prefer in future?

Reply:
We prefer use x::function(), because:
If you use namespace in two files and you will have same function it will not cause a problem, but if you will have a misspell you will have error. If you use :: than you everytime know that you have only misspell.